05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,10 +1,10 @@
 .parent-container {
   display: flex;
-  margin-top: 0;
-  margin-right: auto;
-  margin-bottom:auto;
-  margin-left: 0;
-}
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin-top: 20px;
+  }
```

Comment:
Be sure to push the closing curly braces over to the far left. This helps improve code readability.